CONTRIBUTING.md

```diff
@@ -91,15 +91,54 @@ it will create a new folder inside `packages` with a simple boilerplate for your
 
 ### Prerelease
 
-- Every merged PR should trigger an automatic canary release
+- Every commit or PR merged to the master branch will trigger the Github [prerelease](https://github.com/Turfjs/turf/blob/master/.github/workflows/prerelease.yml) action, creating an alpha release (e.g. `7.0.0-alpha.116` where 116 is the number of commits to master since the last release tag).
 - The version number is calculated by a combination of the output of `git describe` and the `publish:prerelease` script in the root package.json. Specifically having `major` is appropriate for the pre-7.0 development phase, but we should change it to `minor` after 7.0.0 is finally released.
 
 ### Release
-- If necessary, make and merge a PR to contain any last minute changelog entries or other housekeeping items
-- Make a PR that contains the file changes from `pnpm lerna version --no-push 7.0.0` (update the version as necessary)
-- When pushing the release PR, use `--follow-tags` to make sure the relevant tag is also pushed
-- The CI will start doing the release when the tag is created
-- Don't forget to merge your PR
+- If necessary, make and merge a PR with any last minute housekeeping items (docs, etc.)
+- Review code commits and decide the new version number.  If there are breaking changes, then it should be a major version bump, e.g. 6.x.x to 7.0.0.  This project follows [semantic versioning](https://semver.org/).
+
+#### Release commands
+Run the following release commands, replacing 7.0.0 with the version number to release:
+
+- fetch the latest code from remote origin
+```bash
+git fetch origin
+```
+
+- create a release branch, replace mf with your initials to make it clear whose branch it is.
+```bash
+get checkout origin/master -b mf/release-7.0.0
+```
+
+- increment the version number of all packages, without pushing to origin.  This will also create a release tag.
+```bash
+pnpm lerna version --no-push 7.0.0
+```
+
+- check the changed files and stage them for commit.  Example:
+```bash
+git add packages/*/package.json
+```
+
+- create new commit on the release branch
+```bash
+git commit -m "Release v7.0.0"
+```
+
+- push the release branch and the release tag.  This will trigger the Github [release](https://github.com/Turfjs/turf/blob/master/.github/workflows/release.yml) action.
+```bash
+git push origin mf/release-7.0.0 --follow-tags
+```
+
+#### Release Final Steps
+- Create a Pull Request for the release, using the link in the output of the push command and the version number for the PR title (e.g. v7.0.0).  If the link isn't provided, just create a PR setup to merge your release branch to master.  Don't merge the PR yet.  Here is an example - https://github.com/Turfjs/turf/pull/2615
+
+- You can view the status of the triggered release action here - https://github.com/Turfjs/turf/actions.  Once complete, a new [version](https://www.npmjs.com/package/@turf/turf?activeTab=versions) of all turf packages will have been published on NPM.
```

Comment:
This paragraph probably belongs at the end of the prev section after `push ... --follow-tags`. If I understand that's what kicks off the release action.

Reply:
Done.  I actually merged the two bullets together